src/ovirtapi/index.js

```diff
@@ -1,6 +1,9 @@
 // @flow
 import type {
-  CdRomType, ApiCdRomType,
+  CdRomType,
```

Comment:
Can you, please, pair types, like Scott had it, I think it would be easier to understand?
```
import type {
  CdRomType, ApiCdRomType,
  SnapshotType, ApiSnapshotType,
} from './types'
```

Reply:
done